pulp_container/app/viewsets.py

```diff
@@ -696,6 +696,7 @@ class ContainerRepositoryViewSet(
                 "condition": [
                     "has_model_or_obj_perms:container.build_image_containerrepository",
                     "has_model_or_obj_perms:container.view_containerrepository",
+                    "has_repo_or_repo_ver_param_model_or_obj_perms:file.view_filerepository",
```

Comment:
Or maybe these functions _do_ allow to pass parameters.
(It would need a change in pulpcore first, right?)